test/quantization/core/test_quantized_op.py

```diff
@@ -3814,6 +3814,111 @@ def test_qlinear_unpack(self, W, use_channelwise):
             np.testing.assert_equal(
                 W_q.q_zero_point(), W_q_origin.q_zero_point())
 
+    @given(batch_size=st.integers(1, 4),
+           input_channels=st.integers(16, 32),
+           output_channels=st.integers(4, 8),
+           use_bias=st.booleans(),
+           use_multi_dim_input=st.booleans(),
+           use_channelwise=st.booleans())
```

Comment:
we can use this locally to make sure the implementation is good, but ideally we don't want to commit them to master, hypothesis tests has caused many issues in the past since it's not deterministic and always tries to find corner cases that the test fail, and typically it can find some since quantized kernels has rounding, clamping etc. which makes it not very numerically stable.